apps/messaging-api/routes/providers/emails.ts

```diff
@@ -58,16 +66,23 @@ export default async function emails(app: FastifyInstance) {
           200: Type.Object({
             data: Type.Array(EmailProviderType),
           }),
-          "4xx": { $ref: "HttpError" },
-          "5xx": { $ref: "HttpError" },
+          "4xx": HttpError,
+          "5xx": HttpError,
         },
       },
     },
     async function handler() {
-      const client = await app.pg.connect();
+      const client = await app.pg.pool.connect();
       const service = mailService(client);
       try {
-        return { data: await service.getProviders() };
+        return { data: await service.getProviders(organisationId) };
+      } catch (err) {
+        throw new ValidationError(EMAIL_PROVIDER_ERROR, "", [
+          {
+            fieldName: "provider",
+            message: "in_user",
```

Comment:
```suggestion
            message: "alreadyInUse",
```
As you wrote in other cases

Reply:
Thanks, let's use camel to make translations straight forward? Would be nice with some type/const package but let's not worry about that now.

Reply:
Actually forgot to remove this one 😇